README.md

```diff
@@ -1,4 +1,48 @@
-HtUserRegistrationDoctrineORM
-=============================
-
-Doctrine2 ORM storage adapter for HtUserRegistration.
+HtUserRegistrationDoctrineORM
+=============================
+
+This module extends [HtUserRegistration](https://github.com/hrevert/HtUserRegistration)
+by replacing the Zend DB code with Doctrine.
+
+Install
+-------
+
+Require with composer
+```
+composer require hrevert/ht-user-registration-doctrine-orm
+```
+
+Include in config/application.config.php
+```php
+    'modules' => array(
+        ...
+        'HtUserRegistration',
+        'HtUserRegistrationDoctrineORM',  // include after HtUserRegistration
+        ...
+    ),
+```
+
+
+Configure
+----------
+
+Your User entity must implement `HtUserRegistrationDoctrineORM\Entity\UserInterface`.

```

Comment:
In those cases, it is absolutely necessary to implement those.
In this case, I don't think it is absolutely necessary. This module should not enforce to implement the methods, which are not absolutely necessary. If the user/developer wants those methods, he will add those.